src/main/java/qna/domain/Answer.java

```diff
@@ -24,12 +26,14 @@ public class Answer extends AuditingEntity {
     private Long id;
 
     @NotNull
-    @Column(name = "writer_id", updatable = false)
-    private Long writerId;
+    @OneToOne
+    @JoinColumn(name = "writer_id", updatable = false)
```

Comment:
```suggestion
    @JoinColumn(updatable = false)
```

name이 존재하지 않으면 어떻게 동작하는지 알고 계실까요?